src/statick_tool/plugins/tool/dockerfile_lint.py

```diff
@@ -15,18 +15,31 @@ class DockerfileULintToolPlugin(ToolPlugin):
 
     def get_name(self) -> str:
         """Get name of tool."""
-        return "dockerfile-lint"
+        return "dockerfile_lint"
 
```

Comment:
do you think this change will trip anyone up? it'll change what's in our output json. but i like the change.